projects/admin/src/app/circulation/patron/loan/loan.component.ts

```diff
@@ -288,8 +288,9 @@ export class LoanComponent implements OnInit, OnDestroy {
               this.checkedInItems.unshift(newItem);
               // display a toast message if the item goes in transit...
               if (newItem.status === ItemStatus.IN_TRANSIT) {
+                const destination = newItem.loan.item_destination.library_name;
                 this._toastService.warning(
-                  this._translateService.instant('The item is in transit'),
+                  this._translateService.instant('The item is in transit to [{{ destination }}]', {destination}),
```

Comment:
Make this string `The item is in transit to [{{ destination }}` into the file `manual_translations`